src/Badge/Service/ImageFactory.php

```diff
@@ -24,6 +24,7 @@ public function createFromBadge(Badge $badge): Image
             $badge->getSubject(),
             $badge->getStatus(),
             \trim($badge->getHexColor(), '#'),
+            'flat',
```

Comment:
you should use `PUGX\Poser\Badge::DEFAULT_STYLE`?

Reply:
done